src/Cli/dotnet/CommonOptions.cs

```diff
@@ -272,6 +272,41 @@ private static IEnumerable<string> ForwardSelfContainedOptions(bool isSelfContai
             parseResult.HasOption(ArchitectureOption) ||
             parseResult.HasOption(LongFormArchitectureOption) ||
             parseResult.HasOption(OperatingSystemOption);
+
+        internal static string BuildProperty<TValue>(string name, TValue value, bool escapeValue = false)
```

Comment:
escape or escaped?

Reply:
I would say `escape` because that's what we want to do not what's done before.

Reply:
Thanks! There was indeed a problem between what I was saying and the implemented condition!